.travis.yml

```diff
@@ -1,24 +1,37 @@
 language: node_js
 
+sudo: false
+
 node_js:
-  - "0.10"
+  - "4"
   - "0.12"
-  - "iojs-v1"
-  - "iojs-v2"
-
-sudo: false
 
 cache:
   directories:
     - node_modules
 
 env:
-  - DIALECT=mysql
-  - DIALECT=postgres
-  - DIALECT=postgres-native
-  - DIALECT=sqlite
-  - DIALECT=mariadb
-  - DIALECT=mssql
+  global:
+    - CC=clang
+    - CXX=clang++
+    - npm_config_clang=1
+    - COVERALLS_REPO_TOKEN=751SiA6QhY5EH4P2ygq7JFWulqiD2ETXO
+    - COVERALS_PARALLEL=true
```

Comment:
Hmm, this should be the variable that tells coveralls to expect multiple coverage reports - but seems to work fine without it https://coveralls.io/builds/6236971